setup.py

```diff
@@ -72,6 +72,18 @@
     show_help
 )
 
+
+import os
+if os.environ.get('READTHEDOCS') == 'True':
+    # Workaround to upgrade ``npm`` since Read the Docs build environment have
+    # ``3.5.2`` and we need at least ``npm>=6.0``
+    os.system('cd ~ ; mkdir bin ; npm install npm')
+    os.environ['PATH'] = '/home/docs/node_modules/.bin/:{}'.format(os.environ.get('PATH'))
+    print('PATH: {}'.format(os.environ.get('PATH')))
```

Comment:
Let's put all this in a function in `_setup_helpers.py`, e.g. `fixup_npm_for_rtd()`